templates/opensshd.conf.j2

```diff
@@ -275,3 +240,4 @@ Match User {{ item.user }}
     {{ item.rules | indent(4) }}
 {% endfor %}
 {% endif %}
+
```

Comment:
Why add a trailing empty line :) ? This is usually unneeded. Maybe you will find https://github.com/ypid/dotfiles/blob/33ec82ad4bf59409a37891086c765e58c06713a6/vimrc#L888-L898 useful depending on your editor.

Reply:
Fixed, thanks.